frontend-service/src/components/EditProfileModal.js

```diff
@@ -0,0 +1,70 @@
+import axios from "axios";
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import { useUser, useUserDispatch } from "../contexts/UserContext";
+import Avatar from "@mui/material/Avatar";
+import Stack from "@mui/material/Stack";
+import Card from "@mui/material/Card";
+import Button from "@mui/material/Button";
+import TextField from "@mui/material/TextField";
+import Typography from "@mui/material/Typography";
+import Modal from "@mui/material/Modal";
+
+const EditProfileModal = ({isModalOpen, setIsModalOpen}) => {
+  const navigate = useNavigate();
+  const user = useUser();
+  const dispatch = useUserDispatch();
+
+  const [inputName, setInputName] = useState(user.name);
+  const handleCloseModal = () => setIsModalOpen(false);
+  const handleUpdateName = async (event) => {
+    event.preventDefault();
+    const token = window.localStorage.getItem("token");
+    try {
+        const response = await axios.put(
```

Comment:
```suggestion
        await axios.put(
```

You are not doing anything with `response` right?